cmd/keeper/cmd/keeper.go

```diff
@@ -456,6 +462,9 @@ type PostgresKeeper struct {
 	lastPGState     *cluster.PostgresState
 
 	waitSyncStandbysSynced bool
+
+	neverMaster             bool
+	neverSynchronousReplica bool
```

Comment:
Same here (since I don't lilke negated flags). What about `canBeMaster` `canBeSynchronousReplica` that defaults to true? The unique issue is that if you update the sentinel but not the keeper it'll have `canBeMaster` as false so we should make it a `*bool` and default it to true when nil.